activerecord-instrumentation.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler", "~> 1.17"
-  spec.add_development_dependency "dox-style"
+  # spec.add_development_dependency "dox-style"
```

Comment:
private gem

Reply:
We may as well remove the commented dox-style reference entirely